src/Xrefcheck/Core.hs

```diff
@@ -102,26 +102,36 @@ data Anchor = Anchor
   deriving stock (Show, Eq, Generic)
   deriving anyclass NFData
 
+data CopyPaste = CopyPaste
+  { cpAnchorText :: Text
+  , cpPlainText  :: Text
+  , cpPosition   :: Position
+  }
+  deriving stock (Show, Eq, Generic)
+  deriving anyclass NFData
+
 data FileInfoDiff = FileInfoDiff
   { _fidReferences :: DList Reference
   , _fidAnchors    :: DList Anchor
+  , _fidCopyPastes :: DList CopyPaste
   }
 makeLenses ''FileInfoDiff
 
 diffToFileInfo :: FileInfoDiff -> FileInfo
-diffToFileInfo (FileInfoDiff refs anchors) =
-    FileInfo (DList.toList refs) (DList.toList anchors)
+diffToFileInfo (FileInfoDiff refs anchors pastas) =
```

Comment:
**Nitpick**: I believe here should be `pastes`

Reply:
:spaghetti: